requirements-test.txt

```diff
@@ -1,9 +1,9 @@
 asgiar==0.2.0
 fakeredis==1.5.0
-kp-registry @ git+https://github.com/ranking-agent/kp_registry@v2.2.0
+kp-registry @ git+https://github.com/ranking-agent/kp_registry
 pytest==6.2.4
 pytest-asyncio==0.15.1
 pytest-cov==2.12.1
-binder @ git+https://github.com/TranslatorSRI/binder@v4.3.2
+binder @ git+https://github.com/TranslatorSRI/binder
```

Comment:
Should we be locking these down?

Reply:
I had a little internal debate here as well. Where I landed was that these were kind of redundant with the lock files sitting right next to these. I figured not pinning them here would help keep Strider using the latest versions of these dependencies (as was needed with the kp-registry, which was two minor versions behind).

Reply:
Cool. You thought about it and I like that.